src/reports/components/report-sections/report-body.tsx

```diff
@@ -38,7 +39,9 @@ export const ReportBody = NamedFC<ReportBodyProps>('ReportBody', props => {
                     <NotApplicableChecksSection {...sectionProps} />
                 </ResultsContainer>
             </ContentContainer>
-            <FooterSection {...sectionProps} />
+            <FooterSection {...sectionProps}>
```

Comment:
Since we're not using `environmentInfo` on the `FooterSection` anymore, I'm wondering if we can avoid passing `sectionProps` here.

Reply:
Yes